sdk/spring/azure-spring-boot-starter-active-directory/README.md

```diff
@@ -284,14 +266,6 @@ Please refer to [azure-spring-boot-sample-active-directory-spring-oauth2-resourc
   </dependencies>
 ```
 
-####  Configure application.properties:
-```properties
-azure.activedirectory.app-id-uri=xxxxxxxx-app-id-uri-xxxxxxxxxx
-azure.activedirectory.session-stateless=true
-#Use a port that is not occupied
-server.port=8081
```

Comment:
Sorry for didn't make it clear last time, I mean the configuration `server.port=8081` is not required last time

Reply:
okay